package.json

```diff
@@ -94,6 +94,6 @@
 		}
 	},
 	"dependencies": {
-		"@sveltejs/vite-plugin-svelte": "^3.0.2"
+		"@sveltejs/vite-plugin-svelte": "^4.0.0"
```

Comment:
why is vps a dependency here, should be a peerDependency if its really needed, but i thought this was runtime only?

Reply:
It's definitely a mistake...should've always been in devDeps (possibly in the peers). I don't think i ever touched this so it's probably a byproduct of me generating this lib project a long time ago. Let me move it to devDeps

Reply:
@dominikg i didn't add `@sveltejs/vite-plugin-svelte` because it already has a peerdep on kit and svelte...do you think it should also be a peerdep (maybe specifying ^4.0.0?

Reply:
your package does not depend on vps itself, does it? the peer dependency on sveltekit is because it uses its page store but you don't have to repeat a transitive peer dep unless you need it yourself